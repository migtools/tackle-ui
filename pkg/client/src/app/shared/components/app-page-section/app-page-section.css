.pageSection {
  padding-bottom: 0px !important;
}
