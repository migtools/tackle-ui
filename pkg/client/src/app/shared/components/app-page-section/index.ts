export { AppPageSection } from "./app-page-section";
