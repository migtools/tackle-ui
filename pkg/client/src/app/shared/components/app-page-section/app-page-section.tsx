import React from "react";
import { PageSection, PageSectionVariants } from "@patternfly/react-core";

import "./app-page-section.css";

export const AppPageSection: React.FC = ({ children }) => {
  return (
    <PageSection variant={PageSectionVariants.light} className="pageSection">
      {children}
    </PageSection>
  );
};
