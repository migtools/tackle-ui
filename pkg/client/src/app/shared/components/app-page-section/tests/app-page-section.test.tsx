import React from "react";
import { shallow } from "enzyme";
import { AppPageSection } from "../app-page-section";

describe("AppPageSection", () => {
  it("Renders without crashing", () => {
    const wrapper = shallow(<AppPageSection />);
    expect(wrapper).toMatchSnapshot();
  });
});
