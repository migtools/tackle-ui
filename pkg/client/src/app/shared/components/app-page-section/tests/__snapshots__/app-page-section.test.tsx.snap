// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppPageSection Renders without crashing 1`] = `
<PageSection
  className="pageSection"
  variant="light"
/>
`;
